### PR TITLE
chore:Add new link to c++(#2040)

### LINKED ISSUE #2040: [Add] New link for C++ in Languages category
### Site URL

https://youtu.be/GQp1zzTwrIg?si=40MAUhhef8A8Ft1R

### Description of the above provided site

It is a full C++ programming course. It consists of many lectures whose goal is to take you from beginner to advanced programming level.

### Category

Languages

### Sub Category

C++ Programming

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am participating in Hacktoberfest 2023

### PR DESCRIPTION
**Pull request by ankush-w17**
## Fixes Issue
Closes #2040 

## Changes proposed
Added new c++ course to the category : language sub-category : c++

## Screenshots

Screenshot 2023-10-20 181905

### CODE REVIEW

File: database/languages/cpp.json [hunk 1]
```diff
@@ -33,5 +33,12 @@
     "url": "https://www.youtube.com/watch?v=RRVYpIET_RU",
     "category": "languages",
     "subcategory": "cpp"
+  },
+    {
+    "name": "C++ FULL COURSE For Beginners (Learn C++ in 10 hours)",
+    "description": "It is a full C++ programming course. It consists of many lectures whose goal is to take you from beginner to advanced programming level.",
```
Comment (rupali-codes) [inline database/languages/cpp.json:39]:
```suggestion
    "description": "It's a complete C++ tutorials. It convers all of the fundamental topics of the C++ language.",
```

**Review by Anmol-Baranwal — DISMISSED** [review]
🚀 Looks good to me.

File: database/languages/cpp.json [hunk 2]
```diff
@@ -33,5 +33,12 @@
     "url": "https://www.youtube.com/watch?v=RRVYpIET_RU",
     "category": "languages",
     "subcategory": "cpp"
+  },
+    {
+    "name": "C++ FULL COURSE For Beginners (Learn C++ in 10 hours)",
+    "description": "It's a complete C++ tutorials. It convers all of the fundamental topics of the C++ language.",
```
Comment (CBID2) [inline database/languages/cpp.json:39]:
```suggestion
    "description": "This video covers all of the fundamental topics of the C++ language.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @ankush-w17! :) I pointed out some [typos or proofreading errors] that could be improved. Other than that, you’re off to a great start! :)

**Review by CBID2 — APPROVED** [review]
Looks good to me! :)

**Review by Anmol-Baranwal — APPROVED** [review]
🚀 Looks good to me.

**Review by rupali-codes — APPROVED** [review]
Lgtm

**Pull request merged**